app/models/manga.rb

```diff
@@ -86,16 +86,31 @@ def canonical_title
     romaji_title
   end
 
-  def poster_image_thumb
+  def poster_image_thumb(size = :large)
```

Comment:
If this method is identical between `Anime` and `Manga`, it should probably be broken into a concern (especially with these changes adding so many LOC)